crypto/evp/names.c

```diff
@@ -71,6 +72,22 @@ int EVP_add_mac(const EVP_MAC *m)
     return r;
 }
 
+int EVP_add_kdf(const EVP_KDF *k)
```

Comment:
Do we need to do this? (See my other comment about implicit fetch/EVP_KDF_fetch). If we don't support implicit fetch for KDFs then the need for this disappears I think.

Reply:
There still needs to be a table of the kdf_methods held somewhere. Removing this means reputting back in another table which will also be temporary.

Reply:
Does there need to be such a table in libcrypto? Yes we'll need one in the providers, but I don't see why we would need one in libcrypto.

Reply:
To map an nid or name to a method table I believe there has to be a mapping somewhere currently?

Reply:
EVP_KDF/EVP_MAC are not const objects..